.github/env/_build_image.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
For current purpose, e.g. `_vllm_versions.sh` seems more appropriate name.

(Exporting variables does not need to be slowed down by using interactive Bash shell instead of just `/bin/sh`.)

Reply:
fix in [f12616c](https://github.com/opea-project/GenAIComps/pull/1790/commits/f12616ca3b503a52a3522d9042317c6d88839c78)